bin/s2nc.c

```diff
@@ -95,6 +97,11 @@ void usage()
     fprintf(stderr, "    Path to a file where the session ticket can be stored.\n");
     fprintf(stderr, "  --ticket-in [file path]\n");
     fprintf(stderr, "    Path to session ticket file to resume connection.\n");
+    fprintf(stderr, "  --serialize-out [file path]\n");
+    fprintf(stderr, "    Path to a file where a serialized connection can be stored. \n"
+                    "    Note that this feature is intended to be used with our integration test framework and therefore is not expected to work with s2nc alone.\n");
+    fprintf(stderr, "  --deserialize-in [file path]\n");
+    fprintf(stderr, "    Path to a file where a serialized connection lives. Will be used to skip the handshake and start sending encrypted data.\n");
```

Comment:
I think this isn't sufficient documentation of the limits of these options. I think you need a code comment that goes deeper. These options can't do serialization normally, so what ARE they for? What can and can't they test?